project_task_knowledge/views/document_page_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="action_document_page_project_task" model="ir.actions.act_window">
```

Comment:
This action and the other `document.page` action are placed in the wrong file, you should switch them.